app/screens/global_drafts/constants.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export const DRAFT_TYPE_DRAFT = 'draft';
+export const DRAFT_TYPE_SCHEDULED = 'scheduled';
```

Comment:
In a different PR I was commenting that if you add here `as const` to both constants, you will probably have to use less castings.